translate/src/context/Theme.tsx

```diff
@@ -16,36 +16,44 @@ function getSystemTheme() {
 }
 
 export function ThemeProvider({ children }: { children: React.ReactElement }) {
-  const [theme] = useState(() => {
-    const currentTheme = document.body.getAttribute('data-theme') || 'system';
-    if (currentTheme === 'system') {
-      return getSystemTheme();
-    }
-    return currentTheme;
-  });
+  const [theme] = useState(() => document.body.getAttribute('data-theme')!);
```

Comment:
Why is there `!` at the end of the line here and in a couple of places below? There are warning if you run `make eslint`.

Reply:
I did this because of the expectation that the data-theme attribute will always be present on the document.body and thus will not return a null. I used it to bypass the TypeScript type-checking as document.body.getAttribute('data-theme') can potentially return null, and I get an error otherwise. Should I explicitly set a default value instead?

Reply:
Yeah, seems like we'll need the explicit value anyways.